.github/workflows/deploy.yml

```diff
@@ -12,98 +12,50 @@ on:
     types:
```

Comment:
Maybe be easier to see without diffs.
Also, I'd recommend reading the linked Notion doc to understand the infrastructure setup before reading this.